packages/app/src/cli/api/graphql/app_active_version.ts

```diff
@@ -30,7 +30,7 @@ interface AppModuleVersionSpecification {
   identifier: string
   name: string
   options: {
-    managementExperience: 'cli' | 'custom' | 'dashboard'
+    managementExperience: 'cli' | 'custom' | 'dashboard' | 'app_config'
```

Comment:
```suggestion
    managementExperience: 'cli' | 'custom' | 'dashboard' | 'app_config'
```

We'll want to rely on using `experience: :configuration` instead of management experience here for this kinds of app modules

Reply:
@pt2pham , yes, this has been changed in the next PRs but I can remove it here and rebase the changes to them

Reply:
in this PR this change is only used to filter the `configure` extensions in the `deploy` prompt

Reply:
I guess I'm going to leave as it is in this PR and these are the reasons:
- This is [modified](https://github.com/Shopify/cli/pull/3186/files#diff-f06458e98cf1627074d1b1fa874185b69af5918f918da73ca8e19ba4b58ef7aaR35) in the already reviewable [Adapt deploy release prompts](https://github.com/Shopify/cli/pull/3186/) PR which is a follow up of this one
- This PR is working as expected and it can be 🎩  so if it's approve it could be merged for the `versioned app master` branch
- I have split the `prototype` branch to make the PRs easier to review so each of them are dependant among them. This is the first PR so unless the suggestion fixes something that are not solved in the next ones, my recommendation is to simply reference to the place where it's solved
@pt2pham what do you think?

Reply:
Yeah that works for me 🚢